Gravity-tube/src/Components/Modal/Modal.js

```diff
@@ -63,13 +63,12 @@ const onCheckVideoHandler =(playlistId , videos,playlistVideo)=>{
             value={playlistName.title}
           />
         </div>
-     { playlists?.map(({title:{title}, videos, _id: playlistId, isChecked})=>{
-       
+     { playlists?.map(({title:{title}, videos, _id: playlistId,})=>{
        return (
         <div key={playlistId} className="label-box">
         <label htmlFor={playlistId}>
             <input type="checkbox" id={playlistId} 
-            checked ={videos?.find((item=> item._id === videoId ? true : false))}
+            checked = {videos.find((item=> item._id === videoId)) ? true : false}
```

Comment:
Suggestion- since you are using this in multiple place, you could create a util function and use it here.